ERCS/erc-7806.md

```diff
@@ -0,0 +1,451 @@
+---
+eip: 7806
+title: Minimal intent-centric EOA smart account
+description: Minimal effort intent-centric interfaces for EOA account abstraction
+author: hellohanchen (@hellohanchen)
+discussions-to: https://ethereum-magicians.org/t/erc-7806-minimal-intent-centric-eoa-smart-account/21565
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-02
+requires: 7702
+---
+
+## Abstract
+
+This proposal defines interfaces to build intent-centric smart accounts. EOA account can delegate contract code to the
+smart account implementation and sign intents to allow solvers (and relayers) to execute on behalf of account owner.
+
+## Motivation
+
+AA (Account abstraction) is a hot topic in blockchain industry because it gives accounts programmability features,
+including but not limited to:
+
+* **BatchExecution**
+* **GasSponsorship**
+* **AccessControl**
+
+With [ERC-4337](./eip-4337.md), engineers built a permissionless AA standard. While unlocking enormous number of useful
+features, ERC-4337 still has several limitations:
+
+* **Complexity**: Requiring multiple components: Account, EntryPoint, Paymaster, Bundler, and
+  Plugins ([ERC-6900](./eip-6900.md), [ERC-7579](./eip-7579.md). Services like bundler requires high engineer skills and
+  heavy overhead work to run and maintain.
+* **Compatibility**: Compatibility issue between components forces developers to upgrade multiple smart contracts within
+  one version update, also divides the ecosystem.
+* **Cost**: `UserOperation` processing costs high gas units.
+* **Trust**: Although this standard is designed to be permissionless, there are still centralized processes. Paymaster
+  is usually a centralized service because it needs to either trust the account owner to repay the sponsored gas.
+  Bundlers are running on MEV based and accounts need to trust bundler providers to include their user operations in the
+  transaction.
+
+[ERC-7521](./eip-7521.md) discusses an SCA solution with intent-centric design. It enables solvers to fulfill account
+owners' intents by acting "on behalf of" account owners. ERC-7521 allows arbitrary intents and verification logics while
+trying to keep the whole solution forward-compatible.
+
+With `SET_CODE_TX_TYPE=0x04` allowing EOA accounts to set contract code, EOA accounts will gain similar programmability
+as SCA (Smart Contact Account). A new standard, that provides EOA with highly-demanded AA features while potentially
```

Comment:
You should move this expansion "SCA (Smart Contract Account)" to the first time the abbreviation is used (line 42-ish).

Reply:
Updated 👍